platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/internal/cc/impl/problems/ConfigurationCacheProblemsSummary.kt

```diff
@@ -94,23 +99,26 @@ class ConfigurationCacheProblemsSummary(
 
     fun get(): Summary = lock.withLock {
         Summary(
-            totalProblemCount,
-            deferredProblemCount,
-            suppressedSilentlyProblemCount,
-            ImmutableMap.copyOf(problemCauses),
-            ImmutableList.copyOf(originalProblemExceptions),
-            overflowed,
-            maxCollectedProblems,
-            incompatibleTasksCount,
-            incompatibleFeatureCount
+            totalProblemCount = totalProblemCount,
+            uniqueProblemCount = problemCauses.size,
+            deferredProblemCount = deferredProblemCount,
+            consoleProblemCount = totalProblemCount - suppressedSilentlyProblemCount,
+            consoleProblemCauses = ImmutableMap.copyOf(problemCauses.filterValues { it != ProblemSeverity.SuppressedSilently }),
```

Comment:
So is this. Since we only care for problem causes we can show on the console, decided to filter it here, and reflect that in the property name